Core/include/Acts/Definitions/Units.hpp

```diff
@@ -137,14 +137,14 @@ namespace Acts {
 
 namespace UnitConstants {
 // Length, native unit mm
-constexpr double fm = 1e-12;
-constexpr double pm = 1e-9;
-constexpr double um = 1e-3;
-constexpr double nm = 1e-6;
 constexpr double mm = 1.0;
-constexpr double cm = 10.0;
-constexpr double m = 1e3;
-constexpr double km = 1e6;
+constexpr double fm = 1e-12 * mm;
+constexpr double pm = 1e-9 * mm;
+constexpr double nm = 1e-6 * mm;
+constexpr double um = 1e-3 * mm;
+constexpr double cm = 1e1 * mm;
+constexpr double m = 1e3 * mm;
+constexpr double km = 1e6 * mm;
```

Comment:
Is this better than just having them ascend in size?

Reply:
I don't think so but it makes the dependency on `mm` explicit. We also do this for other units so thought it would also make sense here